src/main/java/stats/utils/Resample.java

```diff
@@ -13,18 +14,22 @@
      * by using an random number generator thats uses the given seed
      * This function uses replacement
      */
-    public static <T> void resample(final ISample<T> in, ISample<T> out, int size ,int seed){
```

Comment:
I agree with this implementation. much more to the point

Reply:
The one deleted you mean???

Reply:
no, the one you kept.